.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
   - npm install -g snyk@1.146.0
 
 script:
-  - ./gradlew --no-daemon test
+  - ./gradlew --no-daemon --parallel test
```

Comment:
Are we convinced that parallel and static aren't going to clash?

Reply:
Yep it's fine. Concurrency issues have been fixed now.